### DISCUSSION TITLE
Installation Problem with v1.0.94

### DISCUSSION THREAD

**Discussion by ukmartin** [ventoy/Ventoy#2509] [General]
I have a MS Surface Pro 8 running Windows 11, and I bought a brand-new SanDisk 256GB Flash Drive to install Ventoy onto.

I had read somewhere that Ventoy doesn't like anti-virus software, so the Windows Defender was switched off for installation.

The Flash Drive was originally visible in Explorer, and it seemed to have been formatted with exFAT filing system. I tried installing Ventoy only to see it shoot up to 17%, then fall back to 12%, then hover between 12% and 17% before crashing out.

After this failed attempt at an installation, the Flash Drive disappeared from Explorer. I later saw in Drive Manager that the Volume on the drive had been deleted. I created a new Simple Volume and reformatted again using the SD Card Formatting tool. I tried another installation only for exactly the same sequence of events to happen.

I then got an old Kingston 32 GB Flash Drive, and tried to install onto that. Again, the same things happened.

The log file is attached if that helps
[log.txt](https://github.com/ventoy/Ventoy/files/12233657/log.txt)

I'd appreciate some advice on where things are going wrong and what can be done to complete an installation, please.

**Comment by Roseannepark0211** [comment]
bro,i got same problem with you,have you deal with it?
